shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java

```diff
@@ -62,91 +64,140 @@ private String keyStoreAlias(final String serviceName) {
     @SuppressLint("CommitPrefEdits")
     @Override
     public void clearSecret(final String serviceName, final String key) throws Exception {
-        prefs.edit().remove(sharedPrefKeyPrefix(serviceName) + key).commit();
+        String id = serviceName + ":" + key;
+        NativeLogger.info("KeyStore: clearing secret for " + id);
```

Comment:
@patrickxb are these ids important at all? we can log them safely?

Reply:
They should be fine (should be `keybase:username`), but I just wanted to see the errors so if you want to drop these that's fine with me.